datamade/ccfp-asset-dashboard
# removed unused project attributes from admin site

## Overview
This PR removes unused attributes for the project model in the admin site.

Closes #54.

## Testing Instructions
- Create superuser for the review app: `heroku run python manage.py createsuperuser -a ccfp-asset-d-cleanup-ad-zglu5c`.
- Make sure I hid all of the unneeded project attributes.
